client/allocrunner/test_fixtures/default.conflist.json

```diff
@@ -0,0 +1,44 @@
+{
+	"cniVersion": "0.4.0",
+	"name": "nomad",
+	"plugins": [
+		{
+			"type": "loopback"
+		},
+		{
+			"type": "bridge",
+			"bridge": "",
+			"ipMasq": true,
+			"isGateway": true,
+			"forceAddress": true,
+			"hairpinMode": false,
+			"ipam": {
+				"type": "host-local",
+				"ranges": [
+					[
+						{
+							"subnet": ""
+						}
+					]
```

Comment:
I think the test was called `empty` because it _wasn't_ correctly populated as it will be in the caller, which is how you get this bogus range configuration. It might not be a bad idea to have an actually-default test that populates the minimum set of items we'd expect to see populated.

Reply:
gosh you're right, good catch!

Reply:
I had half a mind to add validation to protect against this, but just stuck with what was here instead to keep it a true no-op refactor.  think I should reach a little further and check for invalid empty values like this?

Reply:
I think it's fine to keep new invalidation logic out of this refactor PR.